src/models/transactions/accountSet.ts

```diff
@@ -0,0 +1,69 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+enum AccountSetFlagEnum {
+    asfRequireDest = 1,
+    asfRequireAuth = 2,
+    asfDisallowXRP = 3,
+    asfDisableMaster = 4,
+    asfAccountTxnID = 5,
+    asfNoFreeze = 6,
+    asfGlobalFreeze = 7,
+    asfDefaultRipple = 8,
+    asfDepositAuth = 9,
+}
+
+export interface AccountSet extends BaseTransaction {
+    TransactionType: "AccountSet";
+    ClearFlag?: number;
+    Domain?: string;
+    EmailHash?: string;
+    MessageKey?: string;
+    SetFlag?: AccountSetFlagEnum;
+    TransferRate?: number;
+    TickSize?: number;
+}
+
+/**
+ * Verify the form and type of an AccountSet at runtime.
+ * 
+ * @param tx - An AccountSet Transaction
+ * @returns - Void.
+ * @throws - When the AccountSet is Malformed.
+ */
+ export function verifyAccountSet(tx: AccountSet): void {
```

Comment:
Can you specify in the jsdoc that this is an optional function to call for JS users?

Reply:
Also I think it makes more sense to have return type be `boolean` instead of `void`. We can return true at the end of the function.

Reply:
The other `verify` methods all return `void` and just raise errors if there's a problem, we should be consistent